plugins/modules/podman_container_copy.py

```diff
@@ -0,0 +1,131 @@
+  - Alessandro Rossi (@kubealex)
+short_description: Copy file into running container
+notes:
+  - Podman may required elevated privileges in order to run properly.
+description:
+  - Copy file or folder from the host to a container and vice-versa.
+options:
+  executable:
+    description:
+      - Path to C(podman) executable if it is not in the C($PATH) on the machine running C(podman)
+    default: 'podman'
+    type: str
+  src:
+    description:
+      - Path of the file/folder to copy from/to the container
+    type: str
+    required: True
+  dest:
+    description:
+      - Path of the destination file/folder to copy from/to the container
+    required: True
+    type: str
+  container:
+    description:
+      - Name/ID of the container to copy from/to
+    required: True
+    type: str
+  from_container:
+    description:
+      - Specify whether or not the file must be copied from the container to the host
+    required: False
+    default: False
+    type: bool
+  archive:
+    description:
+      - Chown copied files to the primary uid/gid of the destination container.
+    required: False
+    default: True
+    type: bool
+  overwrite:
+    description:
+      - Allow to overwrite directories with non-directories and vice versa
+    required: False
+    default: False
+    type: bool
+'''
+
+EXAMPLES = r"""
+- name: Copy file "test.yml" on the host to the "apache" container's root folder
+  containers.podman.podman_search:
+    src: test.yml
+    dest: /
+    container: apache
+- name: Copy file "test.yml" in the "apache" container's root folder to the playbook's folder
+  containers.podman.podman_search:
+    src: /test.yml
+    dest: ./
+    container: apache
+    from_container: True
+"""
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+def copy_file(module, executable, src, dest, container, from_container, archive, overwrite):
+    if from_container:
+        command = [executable, 'cp', '{0}:{1}'.format(container, src), dest]
+    else:
+        command = [executable, 'cp', src, '{0}:{1}'.format(container, dest)]
+
+    if not archive:
+        command.append('--archive=False')
+
+    if overwrite:
+        command.append('--overwrite')
+
+    rc, out, err = module.run_command(command)
+
+    if rc != 0:
+        module.fail_json(msg='Unable to copy file to/from container - {out}'.format(out=err))
+    else:
+        changed = True
+    return changed, out, err
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            executable=dict(type='str', default='podman'),
+            src=dict(type='str', required=True),
+            dest=dict(type='str', required=True),
+            container=dict(type='str', required=True),
+            from_container=dict(type='bool', required=False, default=False),
+            archive=dict(type='bool', required=False, default=True),
+            overwrite=dict(type='bool', required=False, default=False)
+        ),
+        supports_check_mode=True,
```

Comment:
I don't think it supports check_mode it in current revision, isn't it?

Reply:
correct, let me change it!